sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -61,6 +64,10 @@ const COE_NOT_FOUND_MESSAGE =
 const FIRST_COE_NOT_COMPLETE = "FIRST_COE_NOT_COMPLETE";
 const FIRST_COE_NOT_COMPLETE_MESSAGE =
   "First disbursement(COE) not complete. Please complete the first disbursement.";
+export const INVALID_TUITION_REMITTANCE_AMOUNT =
+  "INVALID_TUITION_REMITTANCE_AMOUNT";
+const INVALID_TUITION_REMITTANCE_AMOUNT_MESSAGE =
+  "Tuition amount provided should be lesser of EITHER (Actual tuition + Program related costs) OR (Canada grants + Canada Loan + BC Loan).";
```

Comment:
Just checking on the given message against business logic. Indeed the tuition remittance  must be lesser then both (Actual tuition + Program related costs) AND (Canada grants + Canada Loan + BC Loan). 

We pick the min value between  (Actual tuition + Program related costs) and (Canada grants + Canada Loan + BC Loan). So if lesser than one it implicitly is lesser than both.